kytos/core/config.py

```diff
@@ -9,7 +9,7 @@
 from argparse import ArgumentParser, RawDescriptionHelpFormatter
 from configparser import ConfigParser
 
-from kytos.core import __version__
+from kytos.core.metadata import __version__
```

Comment:
Besides that I think that we should think about this more carefully. Looks like `__version__` needs to be a module thing. PEP0440. That is good, but please, let's think on a better solution.